src/DataTypes/DataTypeObject.cpp

```diff
@@ -44,6 +44,11 @@ String DataTypeObject::doGetName() const
     return out.str();
 }
 
+Field DataTypeObject::getDefault() const
+{
+    return Object{{"dummy", 1}};
+}
```

Comment:
Why not empty object? It should be handled correctly, I think.

Reply:
I suppose it can work first, but does not when return empty object: 

```
DB::Exception: Invalid number of rows in Chunk column Object position 3: expected 1, got 0. (LOGICAL_ERROR)   
```

Reply:
Ok, please do it at least consistent with https://github.com/ClickHouse/ClickHouse/blob/98e6a6b09b5256731108aac2ff4808cb1783a99f/src/Columns/ColumnObject.cpp#L1084-L1087

`{"_dummy": 0}`.